### PR TITLE
#146 Created query for fetching organization task, added 2 more tasks in database

### PR DESCRIPTION
**Pull request by linaskitm**

### CODE REVIEW

File: src/schema/typeDefs/organizationTask.graphql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+type OrganizationTask {
+    id: Int!
+    title: String
+    description: String
+    organization: Int!
+    isDone: Boolean
+}
+ type Query {
+    organizationTasks: [OrganizationTask]
+    organizationTask(id:Int): OrganizationTask
```
Comment (ignasposka) [inline src/schema/typeDefs/organizationTask.graphql:10]: Nitpick: missing empty space between `id:` and `Int`.

File: test/organizationTask.graphql.test.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+const url = process.env.TEST_URL || 'http://localhost:8081';
+const request = supertest(url);
+
+describe('OrganizationTasks test', () => {
+    it('Returns organization tasks list', (done) => {
+        request
+            .post('/graphql')
+            .send({
+                query: `{ organizationTasks {
+                        id,
```
Comment (ignasposka) [inline test/organizationTask.graphql.test.ts:14]: You could put those fields to variable inside `test/testFields.ts` and reuse them. See example in other test files.

**Review by ignasposka — APPROVED** [review]
Please fix those two small comments and I can merge then :)

**Comment by linaskitm** [comment]
Thanks for review, I refactored the code and tested.

**Pull request merged**